GenderPayGap.WebUI/Models/Organisation/OrganisationReportsViewModel.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Database;
+using GenderPayGap.Extensions;
+using GenderPayGap.WebUI.Classes.Formatters;
+using GenderPayGap.WebUI.Helpers;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace GenderPayGap.WebUI.Models.Organisation
+{
+    public class OrganisationReportsViewModel
+    {
+
+        [BindNever]
+        public long OrganisationId { get; set; }
+
+        [BindNever]
+        public Dictionary<int, OrganisationScopeForYear> OrganisationScopesByYear { get; set; }
+
+        public OrganisationReportsViewModel(Database.Organisation organisation, List<int> yearsWithDraftReturns)
+        {
+            this.OrganisationId = organisation.OrganisationId;
+            this.OrganisationScopesByYear = GetOrganisationScopes(organisation, yearsWithDraftReturns);
+        }
+
+        private Dictionary<int, OrganisationScopeForYear> GetOrganisationScopes(Database.Organisation organisation, List<int> yearsWithDraftReturns)
+        {
+            // Get list of all available reporting years
+            List<int> reportingYears = ReportingYearsHelper.GetReportingYears();
+            
+            // Create dictionary to store reporting years as keys against OrganisationScopeForYear values
+            Dictionary<int, OrganisationScopeForYear> scopesByYear = new Dictionary<int, OrganisationScopeForYear>();
+            
+            // Iterate over each available reporting year

```

Comment:
There's a few comments here I find overkill - no particular downside to having them but I definitely don't think they're needed and there's a case for deleting them just to keep the lines in the file down